core/kits/documents/tabs/settings-layout.php

```diff
@@ -100,6 +100,16 @@ protected function register_tab_controls() {
 			? esc_html__( 'Gaps', 'elementor' )
 			: esc_html__( 'Widgets Space', 'elementor' );
 
+		$optimized_markup = Plugin::instance()->experiments->is_feature_active( 'e_optimized_markup' );
+
+		$widget_spacing = $optimized_markup
+			? 'content: ""; display:block; height: {{ROW}}{{UNIT}}'
+			: 'margin-block-end: {{ROW}}{{UNIT}}';
+
+		$widget_spacing_selector = $optimized_markup
+			? '.elementor-widget:not(:last-child)::after'
+			: '.elementor-widget:not(:last-child)';
```

Comment:
@hein-obox @rami-elementor I found an issue with `Sections`. The global widget spacing was using `margin` on the main wrapper, and the margin from the Advanced tab on the inner div. But now that the Advanced tab will apply it on the main wrapper as well, the two margins are conflicting. I saw the layout shifts in the plugins-tester. I think I fixed it with an `::after` pseudo element and giving it a `height`. WDYT? If we can't use this, please help with other suggestions, or the Optimized DOM would need the Container as a dependency. Everything works fine in the Container, because it uses the `gap` property and not margin.

Reply:
@louiswol94 
I understand the complexity. 

I am thinking could we leave the sections as is? In that way users could be motivated more to move from Sections to Containers if they value performance. That could be better approach, then to implement potentially risky styling to the Sections.

Wdyt?